src/general/index.ts

```diff
@@ -228,3 +231,57 @@ export function downloadFile(downloadConfig: { file: File | null; type: string;
     console.error('File download failed - ', err);
   }
 }
+
+
+/**
+ * This method sorts an Object with key value pairs on the basis of the values. (Check examples for better understanding)
+ *
+ * @param {SingleLevelObject} obj - Object with key value pairs with single level hierarchy. (Read remarks)
+ * @param {boolean} isDescending - Sort in descending order or not. Defaults to false. Optional argument.
+ *
+ * @remarks
+ * Object should be of a single level. Avoid nested objects or arrays. In case of error, method returns the original object.
+ *
+ * @example
+ * ```
+ * const list = { yellow: 1, blue: 10, red: 5, green: 6, pink: 8 };
+ * const listWrong = { yellow: 1, blue: [ 'I', 'am', 'blue' ], red: 5, green: { i: 'i', am: 'am', green: 'green' }, pink: 8 };
```

Comment:
why this list wrong? Because of array inside object? or object inside object?

Reply:
Yes.. Once you introduce an object in one or all of the keys.. The single level hierarchy breaks

Reply:
Multi level hierarchy will not be sorted properly.